plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/internal/RegistryMessages.properties

```
@@ -22,4 +22,4 @@ project_open_cannot_read_credentials_title = You are about to lose project crede
 project_open_cannot_read_credentials_message = You are about to open an encrypted project: ''{0}''\n\nWarning: if you proceed, your credentials will be permanently deleted. To avoid losing them, please open your project in DBeaver PRO version.
 project_open_cannot_read_credentials_button_text = Open and delete credentials
 project_open_cannot_read_configuration_title = Error opening the project
-project_open_cannot_read_configuration_message = Configuration for project ''{0}'' is encrypted. Please open it in DBeaver PRO.
+project_open_cannot_read_configuration_message = The ''{0}'' project was protected with a password in another edition of DBeaver. In this edition the password does not work, so this project cannot be opened. You can try to open it in <a href="https://dbeaver.com/download/">another DBeaver edition</a>.
```

For me, it's not clear, what is this, because in the end of previous version you're talking about "another edition".
```suggestion
project_open_cannot_read_configuration_message = The ''{0}'' project was protected with a password in another edition of DBeaver. In the current edition the password does not work, so this project cannot be opened. You can try to open it in <a href="https://dbeaver.com/download/">another DBeaver edition</a>.
```